src/taskgraph/parameters.py

```diff
@@ -80,6 +80,8 @@ def _get_defaults(repo_root=None):
     try:
         repo_url = repo.get_url()
         project = repo_url.rsplit("/", 1)[1]
+        if project.endswith(".git"):
+            project = project[:-4]
```

Comment:
I'm glad this problem comes at the same time as `mozilla-repo-urls` got out. https://github.com/taskcluster/taskgraph/pull/107 introduced it. Let's leverage it here. Once that PR is done, we can bump `mozilla-repo-urls` to 0.0.3 and use this piece of code:

```suggestion
        repo_url = repo.get_url()
        parsed_url = parse(repo_url)
        project = parsed_url.repo_name
```

What do you think, @gabrielBusta?

Reply:
I think it's great :) I can bump it in this PR and get rid of this repetitive logic while Im at it

Reply:
Awesome! mozilla-repo-urls 0.0.3 has been published! 🚢 

https://pypi.org/project/mozilla-repo-urls/0.0.3/#files